### ISSUE TITLE
Some icons are not displayed

### ISSUE THREAD

**Issue by JohnB7777** [ianperrin/MMM-NetworkScanner#46]
Hello,

I don't know why but I no all the icons are working.
If I take "phone", it's displayed by the module but no "mobile".
And yet, they are both "solid icons" (because at the beginning, I was thinking that only "solid icons" were working but no "regular icons", for example - but this is no the case).

How can we change that ?

Thanks !

**Comment by ianperrin** [comment]
Curious, both seem to work for me.

What version of MagicMirror are you using?

**Comment by JohnB7777** [comment]
I'm using the last version, the 2.7.0
And I have a lot of icons which are not working

**Comment by ianperrin** [comment]
After you updated to 2.7, did you run the following command from your MagicMirror directory `git pull && npm install`?

Also, do you have other modules in your config? If so, does the problem persist if you remove them from the config.js file (including all the standard/default) modules and only include Network Scanner?

**Comment by JohnB7777** [comment]
Yes, I did this command (it was necessary for update) and yes I have others modules.
But, the question is : if it's other modules which are the cause, why some icons are displayed ?

**Comment by ianperrin** [comment]
I’m not 100% sure, but on my upgraded instance, and a clean install, I can’t replicate the problem so I was hoping to explore what might be causing your issues.

One possibility is that a module may be referencing other versions of FA than the version included with MM. Referencing multiple versions of FA from the same webpage is known to cause unexpected results.

**Comment by JohnB7777** [comment]
Sorry for my late response but what is "FA" ?

**Comment by ianperrin** [comment]
FontAwesome ;)

**Comment by JohnB7777** [comment]
Ok, thanks :)
So you think that maybe I have another module which uses the same font and it could conflict ?

**Comment by simphide** [comment]
Did you find a solution? I have the same problem.

**Comment by JohnB7777** [comment]
No, I didn't find it :/
I always have some icons well displayed and others which are empty squares

**Comment by ianperrin** [comment]
@JohnB7777 / @simphide - could you confirm which version of MagicMirror you are using and let me know what happens if NetworkScanner is the only module in your config?

**Comment by ghost** [comment]
I have the same issue; 'phone-square-alt' especially...
